Rpackage/R/mmit.R

```diff
@@ -0,0 +1,23 @@
+mmit <- structure(function(formula, target.mat, feature.mat, weights = NULL,
```

Comment:
Hey @tdhock! Do you think its better to receive one dataframe with all variables and then use a formula to extract targets and features, or to receive features and targets in separate matrices? I see that in your penalty learning code, you use the second option. Would like to have your opinion. Thanks!

Reply:
It seems like for most datasets (e.g., Iris dataset), the targets and features are in a single matrix. Is this common in R?